frontend/src/containers/SearchTable.jsx

```diff
@@ -105,27 +105,27 @@ export default class SearchTable extends Component {
         this.setState({
             currentPage: newPage
         }, () => this.loadDataForPage(this.state.currentPage));
-    }
+    };
```

Comment:
oh, I wonder if we should add a linter to the project...

Reply:
Good idea. Another PR for that?